modules/core/src/utils/json-loader.js

```diff
@@ -10,6 +10,7 @@ export default {
   // TODO - can we stream process geojson?
   extensions: ['json', 'geojson'],
   mimeTypes: ['application/json', 'application/geo+json'],
+  text: true,
```

Comment:
@ibgreen why is this required now? There are a handful of loaders that accept text input, e.g. CSVLoader, OBJLoader, and they can't parse string either in the latest beta. Can you provide some rationale behind this change? If it's intentional it should be changed across the board and added to upgrade guide.

Reply:
Yes it is intentional but not a very important change.

Rationales:
- As a general principle we prefer not to mutate user supplied objects
- Since we now have invested so heavily in typescript in loaders.gl 3.0 it is less likely that improperly formatted objects are supplied.
- I removed that normalization fixup some time ago and it passed loaders tests.

How to fix:
- We can certainly add back the line that does `text` field auto injection in loader normalization if it is causing problems.
- If this was detected a bit earlier in the test cycle I would make a deeper investigation
- but right now we are at the end of a big push and still have a few bigger things to sort.

Reply:
@morphoix @dryabinin-actionengine Can you add back the deleted code: https://github.com/visgl/loaders.gl/blob/master/modules/core/src/lib/loader-utils/normalize-loader.ts#L50-L52

Reply:
> Since we now have invested so heavily in typescript in loaders.gl 3.0 it is less likely that improperly formatted objects are supplied.

The `text` field is marked optional. It is not present in any of the official loaders, or https://loaders.gl/docs/developer-guide/creating-loaders-and-writers#creating-a-loader-object

Reply:
Yes, with a bit more work this could be fixed. I'd start by trying to investigate why the `text` field is necessary in the first place. The presence of `parseText...` functions should be enough (after all the `text` field is injected based on the presence of those in the first place). But not enough time to get that change in that right now.